src/main/java/uk/gov/hmcts/darts/common/repository/TransformedMediaRepository.java

```diff
@@ -86,4 +86,19 @@ List<TransformedMediaEntity> findTransformedMedia(Integer mediaId,
                                                       String requestedBy,
                                                       OffsetDateTime requestedAtFrom,
                                                       OffsetDateTime requestedAtTo);
+
+
+    @Query(value = """
+        SELECT tm
+            FROM TransformedMediaEntity tm
+            JOIN tm.mediaRequest media
+            JOIN media.hearing hearing
+        WHERE
+            (:transformedMediaId IS NULL OR (:transformedMediaId IS NOT NULL AND tm.id = :transformedMediaId))
+            AND (:hearingIds IS NULL OR (:hearingIds IS NOT NULL AND media.hearing.id in (:hearingIds)))
+            AND (cast(:endAt as TIMESTAMP) IS NULL OR (media.endTime <= :endAt))
+            AND (cast(:startAt as TIMESTAMP) IS NULL OR (media.startTime >= :startAt))
```

Comment:
See comment in `MediaRequestServiceImpl.adminMediaSearch()`, we should be searching on `media.start_ts` and `media.end_ts` rather than `media_request.start_ts` and `media_request.end_ts`.